config/ember-try.js

```diff
@@ -1,6 +1,5 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
I guess this was just a result of running `ember init` after updating Ember CLI.